### PR TITLE
Add support for Electron 25 & 26

### PR DESCRIPTION
**Pull request by GytisCepk**
## Changes
- Extend supported Electron versions with Electron 25 & 26.
- Increase Electron dev dependency version to `^26.0.0`.

No code changes are required since [changed API for 25](https://github.com/electron/electron/blob/main/docs/breaking-changes.md#planned-breaking-api-changes-250) and [changed API for 26](https://github.com/electron/electron/blob/main/docs/breaking-changes.md#planned-breaking-api-changes-260)  aren't used.

### CODE REVIEW

File: common/changes/@itwin/electron-authorization/gytis-Electron-25-support_2023-06-08-14-29.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/electron-authorization",
+      "comment": "Add support for Electron 25",
+      "type": "patch"
```
Comment (aruniverse) [inline common/changes/@itwin/electron-authorization/gytis-Electron-25-support_2023-06-08-14-29.json:6]:
```suggestion
      "type": "minor"
```

**Review by aruniverse — APPROVED** [review]

File: packages/electron/package.json [hunk 2]
```diff
@@ -72,7 +72,7 @@
   },
   "peerDependencies": {
     "@itwin/core-bentley": "^3.3.0 || ^4.0.0",
-    "electron": ">=23.0.0 <25.0.0"
+    "electron": ">=23.0.0 <26.0.0"
```
Comment (ben-polinsky) [inline packages/electron/package.json:75]: Why not include 26? It is now stable.

Comment (aruniverse) [reply packages/electron/package.json:75]: man i can never keep up electron releases

Comment (aruniverse) [reply packages/electron/package.json]: updated; thx for mentioning.

Comment (ben-polinsky) [reply packages/electron/package.json]: Those Teams RSS alerts are good for something hahah

**Pull request merged**